matador/plotting/ir_plotting.py

```diff
@@ -0,0 +1,126 @@
+""" This submodule implements functions useful for plotting
+the results of infrared and Raman spectroscopy calculations.
+
+"""
+
+from matador.plotting.plotting import plotting_function
+
+
+def read_ir_file(seed):
+    """ This function should take the seed filename and
+    read in the IR data, probably into a dictionary.
+
+    Parameters:
+        seed (str): the filename to read.
+
+    Returns:
+        dict: containing IR data
+
+    """
+
+    with open(seed + '.phonon', 'r') as f:
```

Comment:
Should probably check the seedname here, or in the dispersion script above

```suggestion
    if seed.endswith('.phonon'):
         seed = seed.replace('.phonon', '')
    with open(seed + '.phonon', 'r') as f:
```